### ISSUE TITLE
Accessibility - Semantic HTML 

### ISSUE THREAD

**Issue by bretm9** [Chriscastanuela/whats-cookin#30]
Your HTML must be written semantically and ARIA tags should be used (ONLY if needed / appropriate)
